api/urls.py

```diff
@@ -20,6 +21,7 @@
 urlpatterns += [
     path("accounts/", account_list, name="account_list"),
     path("accounts/<int:pk>/", account_detail, name="account_detail"),
+    path("accounts/create/", AccountCreate.as_view()),
```

Comment:
This isn't a very RESTful url-- in my view, it would be better to create an AccountList class that handles POST and GET, where the get functionality is copied from the existing "account_list" function view. That way we can keep the urls nice and RESTy. Thoughts?

Reply:
Done. Moved both to a class based view, that uses the serializer.